ouroboros-consensus/src/Ouroboros/Consensus/Ledger/Basics.hs

```diff
@@ -688,6 +691,37 @@ calculateAdditions               after = zipOverLedgerTables       (flip rawCalc
 calculateDifference       before after = zipOverLedgerTables       (flip rawCalculateDifference) after (projectLedgerTablesTicked before)
 calculateDifferenceTicked before after = zipOverLedgerTablesTicked (flip rawCalculateDifference) after (projectLedgerTablesTicked before)
 
+rawAttachDiffs ::
```

Comment:
For all these functions, perhaps we could either:
1. Introduce newtypes to make sure we don't swap the arguments.
2. Adjust these to accomodate to some standard typeclasses (`<>`?)

Reply:
Probably will leave this for cleanup